INSTALL.md

```diff
@@ -15,50 +32,51 @@ Update WSL's software before continuing. If you chose Debian, Ubuntu, or another
 apt-get update && apt-get upgrade
 ```
 
-WSL has its own file system that's not accessible from Windows, but Windows files *are* accessible from WSL. So you're going to want to install pokered within Windows. You'll have to change the **current working directory** every time you open WSL.
+WSL has its own file system that's not accessible from Windows, but Windows files *are* accessible from WSL. So you're going to want to install **pokered** within Windows. You'll have to change the **current working directory** every time you open WSL.
```

Comment:
>WSL has its own file system that's not accessible from Windows

This statement is incorrect. The WSL file system is indeed accessible from windows by navigating to `\\wsl$\<Distro>\` from a Windows Explorer window.

> So you're going to want to install **pokered** within Windows.

I would argue that we should encourage users to install pokered within the WSL environment, and not windows. WSL, especially WSLv2 is known for poor windows directory access times. If you install pokered inside the WSL environment, there will be dramatically faster build times.

Reply:
If you are not familiar with the WSL environment enough to rewrite wsl section with this in mind.. then I can make suggestions with changes when I get some time.

Reply:
> If you are not familiar with the WSL environment enough to rewrite the wsl section with this in mind.. then I can make suggestions with changes when I get some time.

Yes please, I would welcome any improvements you might have in mind. I used Cygwin myself; I have no experience with WSL and can't offer any insight into ways to improve the documentation for using it.

Reply:
@Mord5 I applied some suggestions; please proofread them first before applying.